src/rockstor/system/services.py

```diff
@@ -192,8 +147,15 @@ def service_status(service_name, config=None):
     elif service_name == "sftp":
         # Delegate sshd's sftp subsystem status check to system.ssh.py call.
         return is_sftp_running(return_boolean=False)
-    elif service_name in ("replication", "data-collector", "ztask-daemon"):
-        return superctl(service_name, "status", throw=False)
+    elif service_name == "rockstor-scheduling":
```

Comment:
```suggestion
    elif service_name == "scheduling":
```

@Hooverdan96 we need to transition form the internal DB name of scheduling to our systemd name

Reply:
yes, makes sense, that's where I got confused when moving back and forth between the `scheduling` vs `rockstor-scheduling` - when it was the same for the service name and the internal handling everywhere, it did not need to be transparaent.